vulnerabilities/xxe/index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+define( 'DVWA_WEB_PAGE_TO_ROOT', '../../' );
+require_once DVWA_WEB_PAGE_TO_ROOT . 'dvwa/includes/dvwaPage.inc.php';
+
+dvwaPageStartup( array( 'authenticated', 'phpids' ) );
+
+$page = dvwaPageNewGrab();
+$page[ 'title' ]   = 'XML external identity (XXE)' . $page[ 'title_separator' ].$page[ 'title' ];
+$page[ 'page_id' ] = 'xxe';
+$page[ 'help_button' ]   = 'xxe';
+$page[ 'source_button' ] = 'xxe';
+
+dvwaDatabaseConnect();
+
+$vulnerabilityFile = '';
+switch( $_COOKIE[ 'security' ] ) {
+	case 'low':
+		$vulnerabilityFile = 'low.php';
+		break;
+	case 'medium':
+		$vulnerabilityFile = 'medium.php';
+		break;
+	case 'high':
+		$vulnerabilityFile = 'medium.php';
```

Comment:
Shouldn't this be `high.php`?

Reply:
He hasn't written high or impossible levels.

Reply:
I left it to be at medium. This way regular XXE and OOB XXE can be practised.